cypress/support/core-commands.ts

```diff
@@ -35,15 +35,27 @@ Cypress.Commands.add('typeByDataCy', (dataCy: string, text: string) => {
   cy.typeBy(`[data-cy="${dataCy}"]`, text);
 });
 
+Cypress.Commands.add('selectLoadAll', () => {
+  cy.get('#loading').should('not.exist');
+  cy.get('button').then((buttons) => {
+    for (let i = 0; i <= buttons.length; i++) {
+      const button = buttons[i];
+      if (button?.innerText === 'Load more') {
+        button.click();
+        cy.selectLoadAll();
```

Comment:
Could this cause an infinite loop?

Reply:
Load more shows as long as there are more items to load.
Once all items are loaded, it disappears.